src/main/java/com/hm/achievement/utils/AchievementPlaceholderHook.java

```diff
@@ -25,28 +26,62 @@ public AchievementPlaceholderHook(AdvancedAchievements plugin) {
 
 	@Override
 	public String onPlaceholderRequest(Player p, String identifier) {
```

Comment:
We're starting to have quite a few different Placeholders. Let's create a page in the [Wiki](https://github.com/PyvesB/AdvancedAchievements/wiki) to help users out! I can assist in the pages's creation and/or review it once it's ready. 😉

Reply:
Yes good idea! I will write a page later! ;)

Reply:
I write a simple page here => https://github.com/PyvesB/AdvancedAchievements/wiki/Placeholders

:)

Reply:
Wiki page looks good! 😉

Reply:
Thank you! ^^